code/datums/wires/wires.dm

```diff
@@ -1,331 +1,456 @@
-// Wire datums. Created by Giacomand.
-// Was created to replace a horrible case of copy and pasted code with no care for maintability.
-// Goodbye Door wires, Cyborg wires, Vending Machine wires, Autolathe wires
-// Protolathe wires, APC wires and Camera wires!
-
-#define MAX_FLAG 65535
-
-GLOBAL_LIST_EMPTY(same_wires)
-// 12 colours, if you're adding more than 12 wires then add more colours here
-GLOBAL_LIST_INIT(wireColours, list("red", "blue", "green", "black", "orange", "brown", "gold", "gray", "cyan", "navy", "purple", "pink"))
 
 /datum/wires
+	/// TRUE if the wires will be different every time a new wire datum is created.
+	var/randomize = FALSE
+	/// The atom the wires belong too. For example: an airlock.
+	var/atom/holder
+	/// The holder type; used to make sure that the holder is the correct type.
+	var/holder_type
+	/// The display name for the TGUI window. For example, given the var is "APC"...
+	/// When the TGUI window is opened, "wires" will be appended to it's title, and it would become "APC wires".
```

Comment:
I dont know if the double-lined `///` will work with the doc here.

Reply:
Might not. I think I saw this done on TG in a few places, which *seemed* to suggest it was ok but, I dont know for sure.